### ISSUE TITLE
Add support for specifying a path for archive resources

### ISSUE THREAD

**Issue by waybackarchiver** [go-shiori/obelisk#12]

**Comment by abhijit-hota** [comment]
I believe this is already achieved via the `--output` option. From the `README` (emphasis mine):

> - If `--output` flag is not specified then Obelisk will generate file name for the archive and save it in current working directory.
> - If `--output` flag is set to `-` and there is only one URL to process (either from input file or from CLI arguments) then the default output will be `stdout`.
> - If `--output` flag is specified but there are more than one URL to process, Obelisk will generate file name for the archive, but keep using the directory from the specified output path.
> - If `--output` flag is specified **but it sets to an existing directory, Obelisk will also generate file name for the archive.**

Example usage: 

`obelisk "https://philipptanlak.com/web-frontends-in-go" --output blogs/go-frontend.html`

**Comment by waybackarchiver** [comment]
> I believe this is already achieved via the `--output` option. From the `README` (emphasis mine):
> 
> > * If `--output` flag is not specified then Obelisk will generate file name for the archive and save it in current working directory.
> > * If `--output` flag is set to `-` and there is only one URL to process (either from input file or from CLI arguments) then the default output will be `stdout`.
> > * If `--output` flag is specified but there are more than one URL to process, Obelisk will generate file name for the archive, but keep using the directory from the specified output path.
> > * If `--output` flag is specified **but it sets to an existing directory, Obelisk will also generate file name for the archive.**
> 
> Example usage:
> 
> ```shell
> obelisk "https://philipptanlak.com/web-frontends-in-go" --output blogs/go-frontend.html
> ```

You are right. The `--output` flag is used to archive a single file but separate files with [`Archiver.WrapDirectory`](https://github.com/go-shiori/obelisk/blob/347d73aad05b04136d71b094b64fa600bc575b82/archiver.go#L60).
